### ISSUE TITLE
NTR: Anterior neck swelling (Face & neck)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7825]
New term request Anterior neck swelling (nadea3_190423200408): Including bull neck  UMLS: 2180411

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000176
- label: Anterior neck swelling
- parents: [Abnormal neck morphology (HP:0025668)](https://hpo.jax.org/app/browse/term/HP:0025668)
- pmid: 
- def: Abormal enlargement of the front part of the neck owing to accumulation of fluid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7825
- synonyms:
